src/addondirectory.cpp

```diff
@@ -19,6 +19,8 @@ QString rootAppFolder() {
 #ifdef MVPN_WASM
   // https://wiki.qt.io/Qt_for_WebAssembly#Files_and_local_file_system_access
   return "/";
+#elif defined(UNIT_TEST)
+  return QStandardPaths::writableLocation(QStandardPaths::TempLocation);
```

Comment:
I tried using [QStandardPaths::setTestModeEnabled](https://doc.qt.io/qt-6/qstandardpaths.html#setTestModeEnabled) in the tests/unit/helper.h module instead of doing this, but it did not seem to work.